.github/workflows/build.yaml

```diff
@@ -89,27 +89,19 @@ jobs:
           name: rpm-${{ matrix.shortcut }}
           path: ${{ env.ARTIFACTS_DIR }}
 
-      - name: Install documentation generation dependencies
-        if: ${{ matrix.shortcut == 'cs9' }}
-        run: pip3 install pdoc
-
       - name: Install created sdk
-        if: ${{ matrix.shortcut == 'cs9' }}
+        if: ${{ matrix.shortcut == 'cs8' }}
         run: pip3 install . -U
 
-      - name: Checkout target repository
-        if: ${{ matrix.shortcut == 'cs9' }}
-        uses: actions/checkout@v2
-        with:
-          path: ovirt-engine-sdk
-          ref: 'gh-pages'
-
       - name: Create python documentation
-        if: ${{ matrix.shortcut == 'cs9' }}
```

Comment:
Shouldn't it run only on CS8 as other steps generating doc?

```
if: ${{ matrix.shortcut == 'cs8' }}
```

Reply:
thank you, this is a typo from basing this PR on another PR that switched doc-generation to cs8